app/templates/gulpfile.babel(gulp).js

```diff
@@ -392,7 +395,7 @@ gulp.task('build:client', ['transpile:client', 'styles', 'html'], () => {
 
     let assets = plugins.useref.assets({searchPath: ['client', '.tmp']});
 
-    return gulp.src(paths.mainView)<% if(filters.jade) { %>
```

Comment:
Just pull it into a separate commit. If you know how to rebase, I'd suggest it. If not, just make a new PR.